config/schema/og.schema.yml

```diff
@@ -110,6 +110,9 @@ og.og_role.*:
     group_bundle:
       type: string
       label: 'Group bundle'
+    is_admin:
```

Comment:
@damiankloip This is conceptually changing the logic from:

This is a role that might, or might not have the `administer group` permission. It's just another role like all the others 

into:

This is an admin role. The permissions it has under it are not so important, because the role itself is marked as admin.

I think I prefer the existing solution, because it's taking a similar approach to drupal core, where a role is simply a collection of permissions with some metadata (i.e. the role name).

With this approach we'll now have to check for permission both on the Role level, and on the permissions level.

Thoughts?